PyInstaller/depend/analysis.py

```diff
@@ -100,6 +105,43 @@ def __init__(self, pyi_homepath, user_hook_dirs=None, *args, **kwargs):
             os.path.join(self._homepath, 'PyInstaller', 'loader', 'rthooks.dat')
         )
 
+    def msg(self, *args, **kwargs):
+        self._msg(*args, **kwargs)
+
+    def _msg(self, lvl, s, *args):
```

Comment:
The stack depth needs to be set correctly in order to determine where the call to `msg` came from. This could be determined manually but logging has a facility built in if you set the correct stack depth.